pkg/runner/job_runner.go

```diff
@@ -177,7 +177,7 @@ func (jr *JobRunner) Run(j *job.Job) ([][]*job.Report, []*job.Report, error) {
 						}
 					}
 				}
-			}(j, tl, targets, config.LockTimeout)
+			}(j, tl, targets, config.LockRefreshTimeout)
```

Comment:
can you rename `lockTimeout` to `lockRefreshTimeout` in this lambda's signature so it's obvious there too?